backend/lib/utils.js

```diff
@@ -17,8 +19,37 @@ const bool = (env) => env == "true";
 
 const getCookieToken = (req) => req.cookies.token;
 
-const emailRegEx = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
-const isValidEmail = (email) => emailRegEx.test(email);
+// email validation code from front-end 'src\utils\validators'
+const validateTopLevelDomain = (string) => {
+  for (const tld of tlds) {
+    if (string.endsWith("." + tld)) {
+      return true;
+    }
+  }
+  return false;
```

Comment:
What do you think of applying the [Replace Loop with Pipeline Refactoring](https://refactoring.com/catalog/replaceLoopWithPipeline.html)?

```suggestion
tlds.some(tld => string.endsWith("." + tld))
```